.github/workflows/make_pr_for_downstream_repo.yaml

```diff
@@ -81,10 +81,17 @@ jobs:
             This will surface any issues that can arise from merging the PR in Auspice. To address these issues locally, update the source branch (e.g. with a git rebase).
 
             This message and corresponding commits were automatically created by a GitHub Action from [nextstrain/auspice](https://github.com/nextstrain/auspice).
-          draft: true
-          delete-branch: true
```

Comment:
_non-blocking_

I don't see an equivalence for the `delete-branch` input, but I'm not sure we even used it as intended?

Reply:
From the action's [README](https://github.com/peter-evans/create-pull-request/blob/6ce4eca6b6db0ff4f4d1b542dce50e785446dc27/README.md?plain=1#L146):

> If a pull request exists and new changes on the base branch make the pull request unnecessary (i.e. there is no longer a diff between the pull request branch and the base), the pull request is automatically closed. Additionally, if [`delete-branch`](#delete-branch) is set to `true` the `branch` will be deleted.

Yeah, I don't think we actually used it - we just close the PRs and delete the branches manually, which is fine.